core/src/main/web/app/helpers/core.js

```diff
@@ -1033,6 +1053,27 @@
             callback(result);
           }
         });
+      },
+      colorize: function (theme) {
+        var colorizedElements = $("body, html");
+        var ca = colorizedElements.attr('class');
+        var classes = [];
+        if(ca && ca.length && ca.split){
+          ca = jQuery.trim(ca); /* strip leading and trailing spaces */
+          ca = ca.replace(/\s+/g,' '); /* remove doube spaces */
+          classes = ca.split(' ');
+        }
+
+        var classCount = classes.length;
+        for (var i = 0; i < classCount; i++) {
```

Comment:
We are basically doing a find here. This can be simplified a lot with _.find:

    _.find(classes, function(e) {return e.indexOf(BEAKER_THEME) !== -1};)